openapi/v1.1.0/optimade.json

```diff
@@ -325,10 +325,8 @@
             "required": false,
             "schema": {
               "title": "Page Above",
-              "minimum": 0.0,
-              "type": "integer",
-              "description": "RECOMMENDED for use with _value-based_ pagination: using `page_above`/`page_below` and `page_limit` is RECOMMENDED.\nExample: Fetch up to 100 structures above sort-field value 4000 (in this example, server chooses to fetch results sorted by increasing `id`, so `page_above` value refers to an `id` value): `/structures?page_above=4000&page_limit=100`.",
-              "default": 0
+              "type": "string",
```

Comment:
Why `integer` has been replaced with `string`? Shouldn't this be integer-valued parameter?

Reply:
No, we had this wrong, page_above and below are defined with respect to a database ID.

Reply:
Oh, right! The specification says this refers to database ID, I missed that.

Reply:
So did we :grin: